packages/eslint-plugin-lit-a11y/lib/rules/interactive-supports-focus.js

```diff
@@ -0,0 +1,108 @@
+/**
+ * @fileoverview
+ * @author open-wc
+ */
+
+const includes = require('array-includes');
```

Comment:
We don't need this shim, 
https://node.green/#ES2016-features-Array-prototype-includes

so let's just use `Array#includes` instead, like we're doing here on L#63